libs/designer/src/lib/ui/panel/errorsPanel/errorsPanel.tsx

```diff
@@ -0,0 +1,178 @@
+    for (const node of Object.entries(state.operations.inputParameters) ?? []) {
+      const errors = Object.values(node[1].parameterGroups).flatMap((parameterGroup) =>
+        Object.values(parameterGroup.parameters).flatMap((parameter) => parameter.validationErrors ?? [])
+      );
+      if (errors.length > 0) {
+        validationErrorToShow[node[0]] = errors;
+      }
+    }
+    return validationErrorToShow;
+  });
+  const numInputErrors = useMemo(() => {
+    return Object.values(allInputErrors).reduce((acc, curr) => acc + curr.length, 0);
+  }, [allInputErrors]);
+
+  /// Settings
+
+  const allRawSettingsErrors = useAllSettingsValidationErrors();
+  const allSettingsErrors: Record<string, string[]> = useMemo(() => {
+    const validationErrorToShow: Record<string, string[]> = {};
+    for (const [nodeId, v] of Object.entries(allRawSettingsErrors ?? {})) {
+      const errors = v.map((setting) => setting.message ?? '');
+      if (errors.length > 0) validationErrorToShow[nodeId] = errors;
+    }
+    return validationErrorToShow;
+  }, [allRawSettingsErrors]);
+
+  const numSettingsErrors = useMemo(() => {
+    return Object.values(allSettingsErrors).reduce((acc, curr) => acc + curr.length, 0);
+  }, [allSettingsErrors]);
+
+  /// Connections
+
+  const allConnectionErrors = useAllConnectionErrors();
+  const numConnectionErrors = useMemo(() => Object.keys(allConnectionErrors ?? {}).length, [allConnectionErrors]);
+
+  /// Workflow Parameters
+
+  const workflowParameterNames: Record<string, string> = useSelector((state: RootState) =>
+    Object.entries(state.workflowParameters.definitions).reduce((acc: any, curr: any) => {
+      // eslint-disable-next-line no-param-reassign
+      acc[curr[0]] = curr[1].name;
+      return acc;
+    }, {})
+  );
+  const workflowParametersErrors = useWorkflowParameterValidationErrors();
+  const numWorkflowParametersErrors: number = useMemo(() => {
+    return Object.values(workflowParametersErrors ?? {}).reduce((acc, curr) => acc + Object.keys(curr).length, 0);
+  }, [workflowParametersErrors]);
+
+  /// Aggregation
+
+  const allNodesWithErrors = useMemo(
+    () => [
+      ...new Set([
+        ...Object.keys(allInputErrors ?? {}),
+        ...Object.keys(allSettingsErrors ?? {}),
+        ...Object.keys(allConnectionErrors ?? {}),
+      ]),
+    ],
+    [allConnectionErrors, allInputErrors, allSettingsErrors]
+  );
+
+  const numOperationErrors = useMemo(
+    () => numInputErrors + numSettingsErrors + numConnectionErrors,
+    [numInputErrors, numSettingsErrors, numConnectionErrors]
+  );
+
+  const totalNumErrors = useMemo(() => numOperationErrors + numWorkflowParametersErrors, [numOperationErrors, numWorkflowParametersErrors]);
+
+  /// INTL
+
+  const errorPanelHeader = intl.formatMessage({
+    defaultMessage: 'Errors',
+    description: 'Header for the error panel',
+  });
+
+  const noErrorsText = intl.formatMessage({
+    defaultMessage: 'No errors found :-)',
```

Comment:
Do we have guidelines for using emojis? We generally don't include these symbols in UX.

```suggestion
    defaultMessage: 'No errors found.',
```

Reply:
It was just left in from testing, I've removed it